src/main/java/io/confluent/examples/streams/microservices/OrdersService.java

```diff
@@ -248,11 +247,6 @@ private HostStoreInfo getKeyLocationOrBlock(final String id, final AsyncResponse
     return locationOfKey;
   }
 
-  private boolean locationMetadataIsUnavailable(final HostStoreInfo hostWithKey) {
```

Comment:
This method does not work any longer because `NOT_AVAILABLE` is not available any more -- triggered the rewrite from above